wallet-new/src/Cardano/Wallet/Client/Http.hs

```diff
@@ -0,0 +1,22 @@
+module Cardano.Wallet.Client.Http
+    ( module Cardano.Wallet.Client.Http
+      -- * Absract Client export
+    , module Cardano.Wallet.Client
+    ) where
+
+import Universum
+
+import Servant.Client (BaseUrl)
+import Network.HTTP.Client (Manager)
+
+import Cardano.Wallet.Client
+
+-- | Given a 'BaseUrl' and an @http-client@ 'Manager', this returns
+-- a 'WalletClient' that operates in 'IO'.
+mkHttpClient
+    :: BaseUrl
+    -> Manager
+    -> WalletClient IO
+mkHttpClient _baseUrl _manager = error "implement me" -- WalletClient{..}
```

Comment:
We should probably wait for this https://github.com/input-output-hk/cardano-sl/pull/2625 so we don't duplicate things. Yes, I know that's for V0, but we require a couple of mind-blowing functions in order for the client to work (https://github.com/input-output-hk/cardano-sl/pull/2625/files#diff-66efa8b5a7226f01ae4cb3aaee278af2R289).

Reply:
Depending on the time scale, we can possibly just cherry pick those instances for the implementation PR.

I suspect that we'll need more fancy magic like that in the implementation. I can tackle those bits and/or pair with someone to help implementing them if desired.